src/components/Match/matchColumns.jsx

```diff
@@ -50,7 +50,13 @@ export const heroTdColumn = {
 };
 
 const partyStyles = (row, match) => {
-  if (row.party_size === 1) {
+  if (row.lobby_type === 1) { // team matches
```

Comment:
This excludes ranked matches, you should check all players to see if there are any parties.

Reply:
do you mean ranked matches where both sides are a party of 5 players each?

Reply:
lobby_type 1 just means a non-ranked match, it doesn't say anything about whether it's a party match or not.